packages/snapshot-listener/src/index.ts

```diff
@@ -41,7 +41,7 @@ log.info(
 
 const app = express();
 const port = process.env.PORT || DEFAULT_PORT;
-app.use(express.json());
+app.use(express.json);
```

Comment:
Can we really remove these parentheses?

Express.js docs seem to indicate that the `express.json()` call is what returns middleware not necessarily that `express.json` itself is the middleware function since it can take configuration arguments.

Reply:
You are right, this is an issue with @types/express... just bumped the version to fix it